src/engines/html5/html5.js

```diff
@@ -182,6 +188,11 @@ export default class Html5 extends FakeEventTarget implements IEngine {
   attach(): void {
     Object.keys(Html5Events).forEach((html5Event) => {
       this._eventManager.listen(this._el, Html5Events[html5Event], () => {
+        if (Html5Events[html5Event] == Html5Events.PLAYING &&
+          !this._registeredMetadataEvent &&
+          this._config.playback.registerMetadataTrackEvent){
+          setTimeout(()=>{this._registerMetadata()},1000);
```

Comment:
@itaykinnrot why the timeout?

Reply:
On safari the metadata track is visible only in async call after playing - not sure why

Reply:
@itaykinnrot I think we should open a bug to apple on this